libs/policyengine-api/src/policyengine_api/household/user.py

```diff
@@ -3,121 +3,143 @@
 from fastapi.security import HTTPAuthorizationCredentials
 from pydantic import BaseModel
 from sqlmodel import Field, SQLModel, Session
-from policyengine.fastapi.database import SessionGeneratorFactory
-from policyengine.fastapi.auth import JWTDecoder
+from policyengine_fastapi.database import SessionGeneratorFactory
+from policyengine_fastapi.auth import JWTDecoder
```

Comment:
__minor, blocking:__
```suggestion
from policyengine_api.fastapi.database import SessionGeneratorFactory
from policyengine_api.fastapi.auth import JWTDecoder
```